js/jquery.cloudinary.js

```diff
@@ -4293,13 +4296,20 @@ var slice = [].slice,
             if (containerWidth !== 0) {
               switch (false) {
                 case !/w_auto:breakpoints/.test(dataSrc):
-                  requiredWidth = maxWidth(containerWidth, tag);
-                  dataSrc = dataSrc.replace(/w_auto:breakpoints([_0-9]*)(:[0-9]+)?/, "w_auto:breakpoints$1:" + requiredWidth);
+                  if (requiredWidth = maxWidth(containerWidth, tag)) {
```

Comment:
These changes seem unrelated the lazy loading (this solves a different bug if I remember correctly).

Can you please separate the lazyload solution to its own PR so we can understand the changes?

Reply:
Thanks for catching this! I've removed the unrelated change